ansible/files/sonic_lab_serial_link.csv

```diff
@@ -0,0 +1,5 @@
+StartDevice,StartPort,EndDevice,EndPort,BaudRate,FlowControl
+console-2,1,str-7260-10,1,9600,0
```

Comment:
`console-2` is the console server of lab infra. We should use a DUT as StartDevice, please replace with `str-msn2700-01`.